src/ASM/LR/ASMu2Dmx.C

```diff
@@ -850,27 +850,26 @@ bool ASMu2Dmx::evalSolution (Matrix& sField, const Vector& locSol,
   sField.resize(std::accumulate(nc.begin(), nc.end(), 0), nPoints);
   for (size_t i = 0; i < nPoints; i++)
   {
+    std::vector<int>    els;
```

Comment:
Now too many spaces between type and variable  (one is enough since elem_sizes now gone)

Reply:
done.